getting_started.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Getting Started"
+nav_order: 2
+description: >
+    Getting Started with FalkorDB Graph Database.
+---
+
+
+# Getting Started with FalkorDB
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).  
+2. **Python Installed**: Ensure you have Python 3.8+ installed.  
```

Comment:
I think that's obvious, to install the PYTHON client you'll need Python...

Reply:
right, but you need a specific python version

Reply:
It doesn't hurt to mention it